README.md

```diff
@@ -59,6 +59,7 @@ Second, are messages the bot is sending. There are three messages user may want
 - `--message.startup=, [$MESSAGE_STARTUP]` - message sent to the group when bot is started, can be empty
 - `--message.spam=, [$MESSAGE_SPAM]` - message sent to the group when spam detected
 - `--message.dry=, [$MESSAGE_DRY]` - message sent to the group when spam detected in dry mode
+- `--message.restore=, [$MESSAGE_RESTORE]` - message sent to the group when original message is unbanned by admin, can be empty to supress reporting the original message
```

Comment:
We should better highlight that the bot would be the poster of that message, and not the original message forwarded or magically restored.